docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -52,38 +52,81 @@ The LogPipeline resource is managed by the Telemetry Operator, a typical Kuberne
 
 The Telemetry Operator watches all LogPipeline resources and related Secrets. Whenever the configuration changes, it validates the configuration (with a [validating webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)) and generates a new configuration for the Fluent Bit DaemonSet, where several ConfigMaps for the different aspects of the configuration are generated. Furthermore, referenced Secrets are copied into one Secret that is mounted to the DaemonSet as well.
 
-## Set up the LogPipeline
-
-### Step 1: Create a LogPipeline
-
-1. To ship application logs to a new output, create a resource file of kind LogPipeline:
-
-   ```yaml
-   kind: LogPipeline
-   apiVersion: telemetry.kyma-project.io/v1alpha1
-   metadata:
-     name: http-backend
-   spec:
-     output:
-      http:
-        dedot: false
-        format: "json"
-        host:
-          valueFrom:
-            secretKeyRef:
-              name: my-secret
-              namespace: my-namespace
-              key: host
-        port: "80"
-        uri: "/"
-        user:
-          value: "user"
-        password:
-          value: "not-required"
-        tls:
-          disabled: true
-          skipCertificateValidation: true
-   ```
+## Setting up a LogPipeline
+
+### Step 1: Create an Input
+
+1. To create a LogPipeline and define from which resources logs should be included create a LogPipeline with a name and spec with input:
+    ```yaml
+    kind: LogPipeline
+    apiVersion: telemetry.kyma-project.io/v1alpha1
+    metadata:
+      name: http-backend
+    spec:
+      input: ...
+    ```
+
+2. As of now only `application` input can be set, meaning the LogPipeline will collect application logs, which is also the default, when omitting the input altogether.
+    ```yaml
+    spec:
+      input:
+        application: ...
+    ```
+
+3. Selection mechanisms for application logs can be applied on namespace or container level. If nothing is set all namespaces will be considered except the system namespaces `kube-system`, `istio-system`, `kyma-system`, `kyma-integration` which are excluded by default.
+    ```yaml
+    spec:
+      input:
+        application:
+          namespaces: []
+          excludeNamespaces: []
+          containers: []
+          excludeContainers: []
+          includeSystemNamespaces: false
+    ```
+
+Valid input example:
+  ```yaml
+  spec:
+    input:
+      application:
+        includeSystemNamespaces: true
+        excludeContainers:
+          - fluent-bit
+  ```
+
+### Step 2: Create an Output
```

Comment:
```suggestion
### Step 2: Create an output
```
Our guidelines want [sentence case](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology/#capitalization) in headlines.